compose-table/src/main/java/org/dhis2/composetable/ui/DataSetTableUi.kt

```diff
@@ -249,12 +251,29 @@ fun ItemHeader(
                 imageVector = Icons.Outlined.Info,
                 contentDescription = "info",
                 modifier = Modifier
+                    .clickable { displayDescription.value = true }
```

Comment:
why instead of using remember, onClick checks` HeaderDescription != null` and calls a private method to show the `AlertDialog`?. Why do we need the remember?

Reply:
@andresmr, This is because the dialog has been implemented in the table component. I think it would be better to extract the dialog and show it using a callback/interface